ledger_device_rust_sdk/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "ledger_device_rust_sdk"
+version = "0.3.0"
```

Comment:
Did you bumped any of the crate version? Should we?

Reply:
No I did not bump anything, it is just a workspace refactoring and there is no modification in the SDK itself.